docs/FLAVORS_EXTENSION.md

```
@@ -0,0 +1,190 @@
+The generated `FlavorConfig.kt` exposes:
+
+- `VARIANT_NAME`, `BUILD_TYPE`
+- `IS_DEMO`, `IS_PROD`, `IS_DEBUG`, `IS_STAGING`, `IS_RELEASE`
+- `BASE_URL`, `DEMO_USERNAME`, `DEMO_PASSWORD`, `IS_DEMO_BUILD`
+- `ENABLE_LOGGING`, `SHOW_DEBUG_OVERLAY`, `LOG_TAG`
+
+## How to add your own flavors
+
+Create one file in your consumer-app repo:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)